instance/tasks.py

```diff
@@ -39,29 +37,12 @@
 # Tasks #######################################################################
 
 @db_task()
-def provision_instance(instance_pk):
+def provision_instance(instance_ref_id):
     """
     Run provisioning on an existing instance
     """
-    logger.info('Retreiving instance: pk=%s', instance_pk)
-    instance = SingleVMOpenEdXInstance.objects.get(pk=instance_pk)
+    logger.info('Retreiving instance: ID=%s', instance_ref_id)
```

Comment:
@bradenmacdonald Perfect chance to fix this typo: "Retreiving" ⟶ "Retrieving" :)